modules/PreviewAPI/src/main/java/org/gephi/preview/PreviewModelImpl.java

```diff
@@ -82,8 +80,7 @@ Development and Distribution License("CDDL") (collectively, the
     //Properties
     private PreviewProperties properties;
     //Dimensions
-    private Dimension dimensions;
-    private Point topLeftPosition;
+    private Rectangle _graphicsCanvas;
```

Comment:
I put it to try to tell developers not to use this variable directly, `getGraphicsCanvas()` has to be used instead as it defines the variable. Any suggestion for handling this?